pkg/utils/util.go

```diff
@@ -49,7 +49,9 @@ const (
 	// CSI Parameters prefixed with csiParameterPrefix are not passed through
 	// to the driver on CreateSnapshotRequest calls. Instead they are intended
 	// to be used by the CSI external-snapshotter and maybe used to populate
-	// fields in subsequent CSI calls or Kubernetes API objects.
+	// fields in subsequent CSI calls or Kubernetes API objects. An exception
+	// exists for the volume snapshot and volume snapshot content keys, which are
+	// passed as parameters on CreateSnapshotRequest calls.
```

Comment:
I think we should use a different prefix according to the official document:
https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/#dynamic-provisioning
Reusing this prefix does not bring too much value but could easily cause confusion.

Reply:
I think we should be consistent with the prefix we use in external-provisioner:
https://github.com/kubernetes-csi/external-provisioner/blob/0c340861ea33bd03464d978cbc0fd2d14dba251c/pkg/controller/controller.go#L109